sql/parser/eval.go

```diff
@@ -256,6 +304,16 @@ func (d DTuple) Next() Datum {
 	return n
 }
 
+// IsMax implements the Datum interface.
+func (d DTuple) IsMax() bool {
+	return false
+}
+
+// IsMin implements the Datum interface.
+func (d DTuple) IsMin() bool {
+	return false
+}
```

Comment:
Yeah, this is really more of a "not-implemented" false. I'll add some explanatory comments.